docs/content/latest/sample-data/pgexercises.md

```diff
@@ -123,9 +124,7 @@ You are now ready to start working through the [PostgreSQL Exercises](https://pg
 - [String Operations](https://pgexercises.com/questions/string/)
 - [Recursive Queries](https://pgexercises.com/questions/recursive/)
 
-YugabyteDB returns the same results as expected based on the solutions on the PostgreSQL Exercises website, with the following exceptions.
+YugabyteDB returns the same results as expected based on the solutions on the PostgreSQL Exercises website, with the following exception:
 
 - ["Find telephone numbers with parentheses"](https://pgexercises.com/questions/string/reg.html)
   - YugabyteDB returns results with a sort order of strings different than in PostgreSQL due to [hash partitioning in YugabyteDB](../../architecture/docdb-sharding/).
```

Comment:
Is there anything else we want to mention about the string sort order? (Like maybe an explanation?)

```suggestion
YugabyteDB returns the same results as expected based on the solutions on the PostgreSQL Exercises website, with one exception. In the ["Find telephone numbers with parentheses"](https://pgexercises.com/questions/string/reg.html) exercise, YugabyteDB returns results with a sort order of strings different than in PostgreSQL due to [hash partitioning in YugabyteDB](../../architecture/docdb-sharding/).
```